scripts/validate-artifacts.ps1

```diff
@@ -1,121 +1,75 @@
 param(
-    [Parameter(Mandatory=$false, ValueFromPipeline=$true)]
-    [bool]$DownloadArtifacts=$true
+    [Parameter(Mandatory=$true, ValueFromPipeline=$true)]
+    [string]$ArtifactsStorageAccountName='matestorage123'
```

Comment:
The default value for `$ArtifactsStorageAccountName` is set to 'matestorage123'. Ensure this is the intended default or consider removing the default to enforce mandatory input.